pbcommand/models/report.py

```diff
@@ -520,6 +541,17 @@ def _get_attrs_simple(self):
     def _get_attrs_complex_list(self):
         return []
 
+    @staticmethod
+    def merge(columns):
```

Comment:
If you use something like `reduce()`, you can let this operate on 1-length lists. Then, you can avoid if-statements elsewhere.